share/completions/nvim.fish

```diff
@@ -1 +1,59 @@
-complete -c nvim -w vim
+# Options copied from vim.fish
+complete -c nvim -s c -r -d 'Execute Ex command after the first file has been read'
+complete -c nvim -s S -r -d 'Source file after the first file has been read'
+complete -c nvim -l cmd -r -d 'Execute Ex command before loading any vimrc'
+complete -c nvim -s i -r -d 'Set the shada file location'
+complete -c nvim -s o -r -d 'Open horizontally split windows for each file'
+complete -c nvim -s O -r -d 'Open vertically split windows for each file'
+complete -c nvim -s p -r -d 'Open tab pages for each file'
+complete -c nvim -s q -r -d 'Start in quickFix mode'
+complete -c nvim -s r -r -d 'Use swap files for recovery'
+complete -c nvim -s t -xa '(__fish_vim_tags)' -d 'Set the cursor to tag'
+complete -c nvim -s u -r -d 'Use alternative vimrc'
+complete -c nvim -s w -r -d 'Record all typed characters'
+complete -c nvim -s W -r -d 'Record all typed characters (overwrite file)'
+complete -c nvim -s A -d 'Start in Arabic mode'
+complete -c nvim -s b -d 'Start in binary mode'
+complete -c nvim -s d -d 'Start in diff mode'
+complete -c nvim -s D -d 'Debugging mode'
+complete -c nvim -s e -d 'Start in Ex mode, execute stdin as Ex commands'
+complete -c nvim -s E -d 'Start in Ex mode, read stdin as text into buffer 1'
+complete -c nvim -s h -d 'Print help message and exit'
+complete -c nvim -s H -d 'Start in Hebrew mode'
+complete -c nvim -s L -d 'List swap files'
+complete -c nvim -s m -d 'Disable file modification'
+complete -c nvim -s M -d 'Disable buffer modification'
+complete -c nvim -s n -d 'Don\'t use swap files'
+complete -c nvim -s R -d 'Read-only mode'
+complete -c nvim -s r -d 'List swap files'
+complete -c nvim -s V -d 'Start in verbose mode'
+complete -c nvim -s h -l help -d 'Print help message and exit'
+complete -c nvim -l noplugin -d 'Skip loading plugins'
+complete -c nvim -s v -l version -d 'Print version information and exit'
+complete -c nvim -l clean -d 'Factory defaults: skip vimrc, plugins, shada'
+complete -c nvim -l startuptime -r -d 'Write startup timing messages to <file>'
+
+complete -c nvim -l remote -d 'Edit files on NVim server'
+complete -c nvim -l remote-expr -d 'Evaluate expr on NVim server'
+complete -c nvim -l remote-send -d 'Send keys to NVim server'
+complete -c nvim -l remote-silent -d 'Edit files on NVim server'
```

Comment:
Please change "on NVim server" to "on nvim server specified with --server" as it is required to use `--server <server>` with these commands or an error message will be printed to the terminal (and not visible because a new neovim instance will launch in the alternate buffer).

Also, please do not capitalize nvim as NVim. It is either Neovim to refer to it as a proper noun or neovim colloquially and to refer to the executable (latter is preferred).